R/chooseFromList.R

```diff
@@ -121,7 +121,9 @@ chooseFromList <- function(theList, type = "items", userinfo = NULL, addAllPatte
     }
   }
   booleanList[identifier] <- TRUE
-  message("Selected: ", paste(originalList[identifier], collapse = ", "))
+  msgselected <- originalList[identifier]
+  stopafter <- min(which(cumsum(nchar(msgselected)) > 666), length(msgselected))
```

Comment:
```suggestion
  stopafter <- min(which(cumsum(nchar(msgselected)) > getOption("chooseFromListLimit", 666)), length(msgselected))
```
This would let users configure this, and also might be helpful in the rare case that you actually want to see everything